static/app/views/dashboards/datasetConfig/transactions.tsx

```diff
@@ -0,0 +1,166 @@
+import type {MEPState} from 'sentry/utils/performance/contexts/metricsEnhancedSetting';
+import type {OnDemandControlContext} from 'sentry/utils/performance/contexts/onDemandControl';
+import {generateFieldOptions} from 'sentry/views/discover/utils';
+
+import type {Widget, WidgetQuery} from '../types';
+import {DisplayType} from '../types';
+import {eventViewFromWidget} from '../utils';
+import {EventsSearchBar} from '../widgetBuilder/buildSteps/filterResultsStep/eventsSearchBar';
+
+import {type DatasetConfig, handleOrderByReset} from './base';
+import {
+  filterAggregateParams,
+  getTableSortOptions,
+  renderEventIdAsLinkable,
+  renderTraceAsLinkable,
+  transformEventsResponseToTable,
+} from './errorsAndTransactions';
+
+const DEFAULT_WIDGET_QUERY: WidgetQuery = {
+  name: '',
+  fields: ['count()'],
+  columns: [],
+  fieldAliases: [],
+  aggregates: ['count()'],
+  conditions: '',
+  orderby: '-count()',
+};
+
+export type SeriesWithOrdering = [order: number, series: Series];
+
+// TODO: Commented out functions will be given implementations
+// to be able to make events-stats requests
+export const TransactionsConfig: DatasetConfig<
+  EventsStats | MultiSeriesEventsStats,
+  TableData | EventsTableData
+> = {
+  defaultWidgetQuery: DEFAULT_WIDGET_QUERY,
+  enableEquations: true,
+  getCustomFieldRenderer: getCustomEventsFieldRenderer,
+  SearchBar: EventsSearchBar,
+  // filterSeriesSortOptions,
+  // filterYAxisAggregateParams,
+  // filterYAxisOptions,
+  getTableFieldOptions: getEventsTableFieldOptions,
+  // getTimeseriesSortOptions,
+  getTableSortOptions,
+  getGroupByFieldOptions: getEventsTableFieldOptions,
+  handleOrderByReset,
+  supportedDisplayTypes: [DisplayType.TABLE],
+  getTableRequest: (
+    api: Client,
+    _widget: Widget,
+    query: WidgetQuery,
+    organization: Organization,
+    pageFilters: PageFilters,
+    _onDemandControlContext?: OnDemandControlContext,
+    limit?: number,
+    cursor?: string,
+    referrer?: string,
+    _mepSetting?: MEPState | null
+  ) => {
+    return getEventsRequest(
+      api,
+      query,
+      organization,
+      pageFilters,
+      limit,
+      cursor,
+      referrer
+    );
+  },
+  transformTable: transformEventsResponseToTable,
+  filterAggregateParams,
+};
+
+function getEventsTableFieldOptions(
+  organization: Organization,
+  tags?: TagCollection,
+  customMeasurements?: CustomMeasurementCollection
+) {
+  const measurements = getMeasurements();
+
+  return generateFieldOptions({
+    organization,
+    tagKeys: Object.values(tags ?? {}).map(({key}) => key),
+    measurementKeys: Object.values(measurements).map(({key}) => key),
+    spanOperationBreakdownKeys: SPAN_OP_BREAKDOWN_FIELDS,
+    customMeasurements: Object.values(customMeasurements ?? {}).map(
+      ({key, functions}) => ({
+        key,
+        functions,
+      })
+    ),
+    fieldKeys: TRANSACTION_FIELDS,
+  });
+}
+
+function getCustomEventsFieldRenderer(field: string, meta: MetaType) {
```

Comment:
Do we still need to maintain the logic for unparameterized transactions? There was specific logic there to link to discover

Reply:
good catch!